src/plugins/discourse/fetch.js

```diff
@@ -68,6 +68,8 @@ export type Post = {|
   // reply to the first post.
   +replyToPostIndex: number | null,
   +timestampMs: TimestampMs,
+  // user trustLevel
+  +trustLevel: number | null,
```

Comment:
Do we know that this can be null in the incoming data? Since the authorUsername is not nullable, I would guess the trustLevel isn't either.